storybook/storybook-react/src/OrderedList/OrderedList.stories.tsx

```diff
@@ -105,19 +115,17 @@ export const WithoutMarkers: Story = {
     ],
     markers: false,
   },
+  argTypes: {
+    inverseColor: {
+      table: { disable: true },
+    },
+  },
```

Comment:
I'd keep the controls for all stories the same, it's good to show that certain combinations of props don't work imo

Reply:
Hmm… alright.